server/index.js

```diff
@@ -11,14 +11,17 @@ const types = require('./controllers/types')
 const countries = require('./controllers/countries')
 const statuses = require('./controllers/statuses')
 const labels = require('./controllers/labels')
+const updates = require('./controllers/updates')
 
 const app = express()
 
 app.use(cors())
 app.use(bodyParser.json({strict: false}))
 app.param('groupId', require('./middlewares/group-id'))
+app.param('contactName', require('./middlewares/contact-name'))
 
 app.use('/contacts', contacts)
+app.use('/contacts/group/:groupId/name/:contactName/updates', updates)
```

Comment:
Why to use that strange URI ? 
Why not 
'/contacts/:groupId/:contactName/updates' ?